lib/seed_helper.rb

```diff
@@ -0,0 +1,52 @@
+# methods which are used in seeds and migrations
+class SeedHelper
```

Comment:
Main motivation was to reuse the methods in migrations, but adding tests is not a bad idea.